src/app/(proper_react)/(redesign)/PageLink.tsx

```diff
@@ -18,6 +18,7 @@ export type Props = ComponentProps<typeof Link> & {
 export const PageLink = (props: Props) => {
   const { activeClassName, hasTelemetry, ...otherProps } = props;
   const pathName = usePathname();
+  /* c8 ignore next 2 */
```

Comment:
Like in the TabList component, there was only one place (the old Shell) where the `activeClass` stuff was used, it seems, so we should probably remove that here too, instead of leaving dead code in place with a `c8 ignore`.